### PR TITLE
BUG: Fix issue #515 snake case erase

### PR DESCRIPTION
**Pull request by hjmjohnson**
Fix bug identified in #515

## PR Checklist
- [X] No API changes (https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No major design changes

### CODE REVIEW

File: Wrapping/Generators/SwigInterface/igenerator.py [hunk 1]
```diff
@@ -236,24 +236,24 @@ def warn(self, identifier, msg, doWarn=True):
             self.warnings.add((identifier, msg))
             if self.verbose:
                 if self.options.warningError:
-                    print("error(%s): %s" % (str(identifier), msg), file=sys.stderr)
+                    print(f"error({str(identifier)}): {msg}", file=sys.stderr)
```
Comment (dzenanz) [inline Wrapping/Generators/SwigInterface/igenerator.py:239]: Does this impose Python 3.6 requirement on developer machines (for debugging and building wheels) or for everybody (pip installing)?

**Review by dzenanz — COMMENTED** [review]
LGTM, but Matt should take a look too. Also, I feel we should ask for feedback on the forum before bumping required Python version to 3.6.

File: Wrapping/Generators/SwigInterface/igenerator.py (hunk 1 above)
Comment (hjmjohnson) [reply Wrapping/Generators/SwigInterface/igenerator.py:239]: Python 3.6+ is already a requirement. The answer is yes, but it is not a new requirement.

Comment (hjmjohnson) [reply Wrapping/Generators/SwigInterface/igenerator.py:239]: Well, perhaps my memory is wrong here.  I thought that 3.6 was a minimum requirement, but perhaps it's 3.5?

Comment (hjmjohnson) [reply Wrapping/Generators/SwigInterface/igenerator.py:239]: @dzenanz   Discussion posted:  
https://discourse.itk.org/t/set-minimum-python-version-to-3-6/3528

**Review by thewtex — APPROVED** [review]
🎉 🥇 🙏 awesome to have this addressed!! Thank you, Hans!

File: Wrapping/Generators/SwigInterface/igenerator.py (hunk 1 above)
Comment (thewtex) [reply Wrapping/Generators/SwigInterface/igenerator.py:239]: 👍  for moving to 3.6 as a minimum requirement.

**Pull request merged**